workshops/static/css/amy.css

```diff
@@ -93,10 +93,9 @@ body {
 
 /* when using nav-tab (http://getbootstrap.com/javascript/#tabs) move the
    crispy form down a few notches */
-div[role="tabpanel"] form {
+.tab-content {
```

Comment:
This CSS selector is more specific.
`div[role="tabpanel"] form` also added padding above the `Delete` buttons in the sponsorship table.